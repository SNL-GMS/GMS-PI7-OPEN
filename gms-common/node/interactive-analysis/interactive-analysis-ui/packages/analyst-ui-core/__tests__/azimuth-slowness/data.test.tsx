import { eventData } from '../__data__/event-data';
import { signalDetectionsData } from '../__data__/signal-detections-data';
import {
  endTimeSeconds,
  eventIds,
  signalDetectionsIds,
  startTimeSeconds,
  timeBlock,
} from '../__data__/test-util';

describe('Data should be verified', () => {
  describe('FK Data', () => {

    it('should be the correct time block', () => {
      expect(endTimeSeconds - startTimeSeconds)
        .toBeGreaterThanOrEqual(
        timeBlock
      );
    });

    it('TimeInterval is mocked properly', () => {
      expect(endTimeSeconds - startTimeSeconds)
        .toBeGreaterThanOrEqual(
        timeBlock
      );
    });

    it('should have valid signal detection data', () => {
      expect(signalDetectionsData)
        .toMatchSnapshot();
    });

    it('should have valid signalDetectionIds', () => {
      expect(signalDetectionsIds)
        .toMatchSnapshot();
    });

    it('should have valid event data', () => {
      expect(eventData)
        .toMatchSnapshot();
    });

    it('should have valid eventIds', () => {
      expect(eventIds)
        .toMatchSnapshot();
    });

  });
});
