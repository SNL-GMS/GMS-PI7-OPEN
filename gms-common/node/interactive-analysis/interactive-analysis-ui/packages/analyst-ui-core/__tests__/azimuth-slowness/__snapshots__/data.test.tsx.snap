// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data should be verified FK Data should have valid event data 1`] = `
Object {
  "activeAnalysts": Array [],
  "conflictingSdHypIds": Array [],
  "currentEventHypothesis": Object {
    "eventHypothesis": Object {
      "associationsMaxArrivalTime": 1274399905.01,
      "event": Object {
        "id": "186f997b-7d7d-3151-8b4d-5609f7a8f31f",
        "status": "ReadyForRefinement",
      },
      "id": "186f997b-7d7d-3151-8b4d-5609f7a8f31f",
      "locationSolutionSets": Array [],
      "preferredLocationSolution": Object {
        "creationInfo": Object {
          "creationTime": 1551284809.237,
          "creatorId": "1",
          "creatorName": "Chris",
          "creatorType": "Analyst",
          "id": "dd1a0e56-4672-4ff7-8f1f-ba76e483233a",
        },
        "locationSolution": Object {
          "featurePredictions": Array [],
          "id": "186f997b-7d7d-3151-8b4d-5609f7a8f31f",
          "location": Object {
            "depthKm": 0,
            "latitudeDegrees": 67.57425,
            "longitudeDegrees": 33.59468,
            "time": 1274399850.081,
          },
          "locationBehaviors": Array [
            Object {
              "defining": false,
              "featureMeasurementId": "c6d4c116-f3e4-4b5a-affd-e8f003f5a2d4",
              "featurePredictionId": "00000000-0000-0000-0000-000000000000",
              "residual": 1.28,
              "weight": 0.734,
            },
            Object {
              "defining": true,
              "featureMeasurementId": "9f4698cf-dca1-45f7-94d4-cdcb49ee8815",
              "featurePredictionId": "00000000-0000-0000-0000-000000000000",
              "residual": 2.6,
              "weight": 0.734,
            },
            Object {
              "defining": true,
              "featureMeasurementId": "96759b42-2a51-4b3e-9b61-2acf352d5c3f",
              "featurePredictionId": "00000000-0000-0000-0000-000000000000",
              "residual": -0.971,
              "weight": 0.734,
            },
          ],
          "locationRestraint": Object {
            "depthRestraintKm": null,
            "depthRestraintType": "FIXED_AT_SURFACE",
            "latitudeRestraintDegrees": null,
            "latitudeRestraintType": "UNRESTRAINED",
            "longitudeRestraintDegrees": null,
            "longitudeRestraintType": "UNRESTRAINED",
            "timeRestraint": null,
            "timeRestraintType": "UNRESTRAINED",
          },
          "locationType": "standard",
          "locationUncertainty": Object {
            "ellipses": Array [
              Object {
                "confidenceLevel": 0.9,
                "depthUncertainty": -1,
                "kWeight": 0,
                "majorAxisLength": "49.1513",
                "majorAxisTrend": 37.23,
                "minorAxisLength": "29.2083",
                "minorAxisTrend": -1,
                "scalingFactorType": "CONFIDENCE",
                "timeUncertainty": "PT4.235S",
              },
            ],
            "ellipsoids": Array [],
            "stDevOneObservation": 1.0484,
            "tt": 6.6189,
            "xt": -26.6202,
            "xy": 163.665,
            "xz": -1,
            "yt": 20.2312,
            "yy": 400.817,
            "yz": -1,
            "zt": -1,
            "zz": -1,
          },
          "snapshots": Array [],
        },
      },
      "rejected": false,
      "signalDetectionAssociations": Array [
        Object {
          "eventHypothesisId": "186f997b-7d7d-3151-8b4d-5609f7a8f31f",
          "id": "527562d9-027e-4b88-941e-91543763b7a4",
          "rejected": false,
          "signalDetectionHypothesis": Object {
            "creationInfo": Object {
              "creationTime": 1551291953.323,
              "creatorId": "creatorId",
              "creatorName": "Matthew Carrasco",
              "creatorType": "Analyst",
              "id": "1c8f8122-0056-3ed9-9304-ddea79de2393",
            },
            "id": "1c8f8122-0056-3ed9-9304-ddea79de2393",
            "rejected": false,
          },
        },
      ],
    },
    "processingStage": Object {
      "id": "1",
    },
  },
  "hasConflict": false,
  "id": "f43f58f9-6a87-40e3-95ac-44168325fc49",
  "modified": false,
  "status": "ReadyForRefinement",
}
`;

exports[`Data should be verified FK Data should have valid eventIds 1`] = `
Array [
  "f43f58f9-6a87-40e3-95ac-44168325fc49",
]
`;

exports[`Data should be verified FK Data should have valid signal detection data 1`] = `
Array [
  Object {
    "associationModified": false,
    "creationInfo": Object {
      "creationTime": 1544045150.378,
      "creatorId": "creatorId",
      "creatorName": "Matthew Carrasco",
      "creatorType": "Analyst",
      "id": "acbd4ff2-f6e1-4a50-bc43-8ccc4f274b05",
    },
    "currentHypothesis": Object {
      "creationInfo": Object {
        "creationTime": 1544045150.378,
        "creatorId": "creatorId",
        "creatorName": "Matthew Carrasco",
        "creatorType": "Analyst",
        "id": "acbd4ff2-f6e1-4a50-bc43-8ccc4f274b05",
      },
      "featureMeasurements": Array [
        Object {
          "creationInfo": Object {
            "creationTime": 1544045150.376,
            "creatorId": "creatorId",
            "creatorName": "Matthew Carrasco",
            "creatorType": "Analyst",
            "id": "dbdd8420-a448-44a0-9fea-d39fe55b0136",
          },
          "definingRules": Array [
            Object {
              "isDefining": true,
              "operationType": "Location",
            },
          ],
          "featureMeasurementType": "ARRIVAL_TIME",
          "id": "dbdd8420-a448-44a0-9fea-d39fe55b0136",
          "measurementValue": Object {
            "standardDeviation": 0.685,
            "value": 1274393237.85,
          },
        },
        Object {
          "creationInfo": Object {
            "creationTime": 1544045150.376,
            "creatorId": "creatorId",
            "creatorName": "Matthew Carrasco",
            "creatorType": "Analyst",
            "id": "3f9b417b-bc12-44ad-855d-132355708ec5",
          },
          "definingRules": Array [
            Object {
              "isDefining": true,
              "operationType": "Location",
            },
          ],
          "featureMeasurementType": "RECEIVER_TO_SOURCE_AZIMUTH",
          "id": "3f9b417b-bc12-44ad-855d-132355708ec5",
          "measurementValue": Object {
            "measurementValue": 0,
            "referenceTime": 0,
          },
        },
        Object {
          "creationInfo": Object {
            "creationTime": 1544045150.376,
            "creatorId": "creatorId",
            "creatorName": "Matthew Carrasco",
            "creatorType": "Analyst",
            "id": "3f9b417b-bc12-44ad-855d-132355708ec5",
          },
          "definingRules": Array [
            Object {
              "isDefining": true,
              "operationType": "Location",
            },
          ],
          "featureMeasurementType": "SLOWNESS",
          "id": "3f9b417b-bc12-44ad-855d-132355708ec5",
          "measurementValue": Object {
            "measurementValue": 0,
            "referenceTime": 0,
          },
        },
        Object {
          "creationInfo": Object {
            "creationTime": 1544045150.377,
            "creatorId": "creatorId",
            "creatorName": "Matthew Carrasco",
            "creatorType": "Analyst",
            "id": "3f9b417b-bc12-44ad-855d-132355708ec5",
          },
          "definingRules": Array [
            Object {
              "isDefining": true,
              "operationType": "Location",
            },
          ],
          "featureMeasurementType": "PHASE",
          "id": "3f9b417b-bc12-44ad-855d-132355708ec5",
          "measurementValue": Object {
            "confidence": 0,
            "phase": "P",
          },
        },
      ],
      "id": "acbd4ff2-f6e1-4a50-bc43-8ccc4f274b05",
      "rejected": false,
    },
    "hasConflict": false,
    "id": "acbd4ff2-f6e1-4a50-bc43-8ccc4f274b05",
    "modified": false,
    "monitoringOrganization": "TEST",
    "signalDetectionHypothesisHistory": Array [
      Object {
        "arrivalTimeSecs": 1274393334.1,
        "arrivalTimeUncertainty": 0.685,
        "creationInfo": Object {
          "creationTime": 1544048130.971,
          "creatorId": "creatorId",
          "creatorName": "Matthew Carrasco",
          "creatorType": "Analyst",
          "id": "8cdefd4e-803f-48f1-9e51-4229066f9991",
        },
        "id": "b90ee9c8-3fc9-4ccd-95ea-5d1d206c35a7",
        "phase": "tx",
        "rejected": false,
      },
      Object {
        "arrivalTimeSecs": 1274393334.1,
        "arrivalTimeUncertainty": 0.685,
        "creationInfo": Object {
          "creationTime": 1544048130.971,
          "creatorId": "creatorId",
          "creatorName": "Matthew Carrasco",
          "creatorType": "Analyst",
          "id": "e8d95a93-382d-4d84-85ce-983e87bdcb0c",
        },
        "id": "e4ae55cc-a395-471c-8a98-bb2cf08411cd",
        "phase": "P",
        "rejected": false,
      },
    ],
    "station": Object {
      "defaultChannel": Object {
        "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
        "id": "43ed5825-5940-309c-af11-ac49dd6a428a",
        "name": "USA0/SHZ",
        "sampleRate": 40,
      },
      "id": "fd3dbadc-72fd-36c6-a3cc-ca4f7a4f58be",
      "location": Object {
        "elevationKm": 0.17,
        "latDegrees": 44.1998,
        "lonDegrees": 131.9888,
      },
      "name": "USRK",
      "sites": Array [
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "8c33b74e-fb7a-30be-9633-5302fe7cfe9a",
              "name": "USB5/SHZ",
              "sampleRate": 40,
            },
          ],
          "id": "71740262-1a5e-31d8-abba-292acb55bd00",
          "name": "USB5",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "43ed5825-5940-309c-af11-ac49dd6a428a",
              "name": "USA0/SHZ",
              "sampleRate": 40,
            },
          ],
          "id": "b787779d-2b15-32d5-ba50-ea68f996af66",
          "name": "USA0",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "3216dbcf-7c8c-334b-8d0b-5d563d5fe198",
              "name": "USA2/SHZ",
              "sampleRate": 40,
            },
          ],
          "id": "26a90e0f-c3e4-36ea-a043-79df1c0ab4de",
          "name": "USA2",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "d307d9e0-639c-3680-af4d-704a710f920b",
              "name": "USB3/SHZ",
              "sampleRate": 40,
            },
          ],
          "id": "f07b4065-c1a6-3dac-a48f-deae6402a9d8",
          "name": "USB3",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "baada424-8cdb-3141-938d-6df7e35be907",
              "name": "USA3/SHZ",
              "sampleRate": 40,
            },
          ],
          "id": "f16e1417-0610-3fae-afcd-62abc59ecd6b",
          "name": "USA3",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "BROADBAND_HIGH_GAIN_ORTHOGONAL_1",
              "id": "fd55cf8e-e429-3665-8cc2-f75d151a6597",
              "name": "USA0B/BH1",
              "sampleRate": 40,
            },
            Object {
              "channelType": "BROADBAND_HIGH_GAIN_ORTHOGONAL_2",
              "id": "1904ad41-bde3-38cd-95d5-cf4bc04918d3",
              "name": "USA0B/BH2",
              "sampleRate": 40,
            },
            Object {
              "channelType": "BROADBAND_HIGH_GAIN_VERTICAL",
              "id": "194ba49a-dccf-3eef-a804-03e34e029340",
              "name": "USA0B/BHZ",
              "sampleRate": 40,
            },
          ],
          "id": "6955e682-ada3-3259-b65f-cac2c8595768",
          "name": "USA0B",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "452d500f-fa63-308b-99b7-91524bd0d512",
              "name": "USA1/SHZ",
              "sampleRate": 40,
            },
          ],
          "id": "852daf54-5244-35c3-9c49-4904fc0fd2ed",
          "name": "USA1",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "66bdaaae-a959-3c17-bc75-c27a5fde36fa",
              "name": "USB2/SHZ",
              "sampleRate": 40,
            },
          ],
          "id": "bfc2586d-aec4-39d5-a123-aa1256690dd1",
          "name": "USB2",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "1e784eb1-a467-36b5-8c43-9e61506e40cd",
              "name": "USB1/SHZ",
              "sampleRate": 40,
            },
          ],
          "id": "7a16f353-f13f-3c27-a937-ee9685c9b3d3",
          "name": "USB1",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "9bd05262-5637-35ae-8028-cf08f3a0d260",
              "name": "USB4/SHZ",
              "sampleRate": 40,
            },
          ],
          "id": "f078b7a1-cf4f-30e4-8633-ee69f2538f57",
          "name": "USB4",
        },
      ],
    },
  },
  Object {
    "associationModified": false,
    "creationInfo": Object {
      "creationTime": 1544045150.38,
      "creatorId": "creatorId",
      "creatorName": "Matthew Carrasco",
      "creatorType": "Analyst",
      "id": "e4ae55cc-a395-471c-8a98-bb2cf08411cd",
    },
    "currentHypothesis": Object {
      "creationInfo": Object {
        "creationTime": 1544045150.378,
        "creatorId": "creatorId",
        "creatorName": "Matthew Carrasco",
        "creatorType": "Analyst",
        "id": "e4ae55cc-a395-471c-8a98-bb2cf08411cd",
      },
      "featureMeasurements": Array [
        Object {
          "creationInfo": Object {
            "creationTime": 1544045150.378,
            "creatorId": "creatorId",
            "creatorName": "Matthew Carrasco",
            "creatorType": "Analyst",
            "id": "e8d95a93-382d-4d84-85ce-983e87bdcb0c",
          },
          "definingRules": Array [
            Object {
              "isDefining": true,
              "operationType": "Location",
            },
          ],
          "featureMeasurementType": "ARRIVAL_TIME",
          "id": "e8d95a93-382d-4d84-85ce-983e87bdcb0c",
          "measurementValue": Object {
            "standardDeviation": 0.685,
            "value": 1274393334.1,
          },
        },
        Object {
          "creationInfo": Object {
            "creationTime": 1544045150.378,
            "creatorId": "creatorId",
            "creatorName": "Matthew Carrasco",
            "creatorType": "Analyst",
            "id": "8cdefd4e-803f-48f1-9e51-4229066f9991",
          },
          "definingRules": Array [
            Object {
              "isDefining": true,
              "operationType": "Location",
            },
          ],
          "featureMeasurementType": "RECEIVER_TO_SOURCE_AZIMUTH",
          "id": "8cdefd4e-803f-48f1-9e51-4229066f9991",
          "measurementValue": Object {
            "measurementValue": 0,
            "referenceTime": 0,
          },
        },
        Object {
          "creationInfo": Object {
            "creationTime": 1544045150.378,
            "creatorId": "creatorId",
            "creatorName": "Matthew Carrasco",
            "creatorType": "Analyst",
            "id": "8cdefd4e-803f-48f1-9e51-4229066f9991",
          },
          "definingRules": Array [
            Object {
              "isDefining": true,
              "operationType": "Location",
            },
          ],
          "featureMeasurementType": "SLOWNESS",
          "id": "8cdefd4e-803f-48f1-9e51-4229066f9991",
          "measurementValue": Object {
            "measurementValue": 0,
            "referenceTime": 0,
          },
        },
        Object {
          "creationInfo": Object {
            "creationTime": 1544045150.378,
            "creatorId": "creatorId",
            "creatorName": "Matthew Carrasco",
            "creatorType": "Analyst",
            "id": "8cdefd4e-803f-48f1-9e51-4229066f9991",
          },
          "definingRules": Array [
            Object {
              "isDefining": true,
              "operationType": "Location",
            },
          ],
          "featureMeasurementType": "PHASE",
          "id": "8cdefd4e-803f-48f1-9e51-4229066f9991",
          "measurementValue": Object {
            "confidence": 0,
            "phase": "P",
          },
        },
      ],
      "id": "e4ae55cc-a395-471c-8a98-bb2cf08411cd",
      "rejected": false,
    },
    "hasConflict": false,
    "id": "e4ae55cc-a395-471c-8a98-bb2cf08411cd",
    "modified": false,
    "monitoringOrganization": "TEST",
    "signalDetectionHypothesisHistory": Array [
      Object {
        "arrivalTimeSecs": 1274393334.1,
        "arrivalTimeUncertainty": 0.685,
        "creationInfo": Object {
          "creationTime": 1544048130.971,
          "creatorId": "creatorId",
          "creatorName": "Matthew Carrasco",
          "creatorType": "Analyst",
          "id": "8cdefd4e-803f-48f1-9e51-4229066f9991",
        },
        "id": "b90ee9c8-3fc9-4ccd-95ea-5d1d206c35a7",
        "phase": "tx",
        "rejected": false,
      },
      Object {
        "arrivalTimeSecs": 1274393334.1,
        "arrivalTimeUncertainty": 0.685,
        "creationInfo": Object {
          "creationTime": 1544048130.971,
          "creatorId": "creatorId",
          "creatorName": "Matthew Carrasco",
          "creatorType": "Analyst",
          "id": "e8d95a93-382d-4d84-85ce-983e87bdcb0c",
        },
        "id": "e4ae55cc-a395-471c-8a98-bb2cf08411cd",
        "phase": "P",
        "rejected": false,
      },
    ],
    "station": Object {
      "defaultChannel": Object {
        "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
        "id": "955d3b5c-6196-385e-a5c8-faf843b0010c",
        "name": "SONA0/SHZ",
        "sampleRate": 50,
      },
      "id": "7a481f10-e4d3-3687-9efa-622a82eb92cf",
      "location": Object {
        "elevationKm": 1.416,
        "latDegrees": 47.83469,
        "lonDegrees": 106.39499,
      },
      "name": "SONM",
      "sites": Array [
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "90e824d7-dd67-327b-aaf8-a4b5335a1134",
              "name": "SONB2/SHZ",
              "sampleRate": 50,
            },
          ],
          "id": "365d2ac9-8167-3ac0-9251-a1c22a5c1a37",
          "name": "SONB2",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "a42bee67-5565-323e-b9b7-2591136e2797",
              "name": "SONA1/SHZ",
              "sampleRate": 50,
            },
          ],
          "id": "3db24be2-f2d9-37df-9e64-ea6effb9e8b1",
          "name": "SONA1",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "501b6b28-38a3-39ac-bace-697b0017a0b3",
              "name": "SONB3/SHZ",
              "sampleRate": 50,
            },
          ],
          "id": "4468687a-77ca-36a8-b3b9-0ca70c4063ac",
          "name": "SONB3",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "955d3b5c-6196-385e-a5c8-faf843b0010c",
              "name": "SONA0/SHZ",
              "sampleRate": 50,
            },
            Object {
              "channelType": "MID_PERIOD_HIGH_GAIN_NORTH_SOUTH",
              "id": "3eb30be8-a7ea-3699-9143-b5dc1d6eb984",
              "name": "SONA0/MHN",
              "sampleRate": 4,
            },
            Object {
              "channelType": "MID_PERIOD_HIGH_GAIN_EAST_WEST",
              "id": "e8da4855-c17e-3f57-b9ce-e4be5c5b2e05",
              "name": "SONA0/MHE",
              "sampleRate": 4,
            },
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_EAST_WEST",
              "id": "8b70482a-b9b1-3657-b7b6-02ad814e04ba",
              "name": "SONA0/SHE",
              "sampleRate": 50,
            },
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_NORTH_SOUTH",
              "id": "739a0706-129b-365a-ab22-ecdd255d35cb",
              "name": "SONA0/SHN",
              "sampleRate": 50,
            },
            Object {
              "channelType": "MID_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "52b3ee97-9e3d-3d17-8e84-55a2787c8348",
              "name": "SONA0/MHZ",
              "sampleRate": 4,
            },
          ],
          "id": "ceb5ee63-23c6-3714-b54f-31e7dbfe122b",
          "name": "SONA0",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "79640618-04d4-39d3-9149-153d7b1cd5bf",
              "name": "SONA2/SHZ",
              "sampleRate": 50,
            },
          ],
          "id": "ef6aef56-6b1f-3f9c-83f3-b8b0c6acf15f",
          "name": "SONA2",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "3be1380c-03d8-3e6a-b8a9-d6a686cefdb5",
              "name": "SONB5/SHZ",
              "sampleRate": 50,
            },
          ],
          "id": "ae97fcfa-277f-37e6-8d71-17de5b46a817",
          "name": "SONB5",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "20be1156-1871-3269-8d23-9c4d8e39886f",
              "name": "SONA4/SHZ",
              "sampleRate": 50,
            },
          ],
          "id": "11cdb68e-4f36-3e8d-9807-1941472a01b1",
          "name": "SONA4",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "9074806f-8f4b-3acc-92b9-1c37b85d641b",
              "name": "SONB4/SHZ",
              "sampleRate": 50,
            },
          ],
          "id": "c878f3c6-8cca-38d0-9d8a-988d7f861291",
          "name": "SONB4",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "feb255ad-eff0-3655-a2cf-471212b8251f",
              "name": "SONB1/SHZ",
              "sampleRate": 50,
            },
          ],
          "id": "796a2b54-24be-3030-8205-85e7dd3ecd2a",
          "name": "SONB1",
        },
        Object {
          "channels": Array [
            Object {
              "channelType": "SHORT_PERIOD_HIGH_GAIN_VERTICAL",
              "id": "dcf35284-7bbd-3675-9be7-8ab2c0b14a10",
              "name": "SONA3/SHZ",
              "sampleRate": 50,
            },
          ],
          "id": "1b65598f-d50a-310e-aaf7-65ffbdf3fb74",
          "name": "SONA3",
        },
      ],
    },
  },
]
`;

exports[`Data should be verified FK Data should have valid signalDetectionIds 1`] = `
Array [
  "acbd4ff2-f6e1-4a50-bc43-8ccc4f274b05",
  "e4ae55cc-a395-471c-8a98-bb2cf08411cd",
]
`;
